edgelet/iotedge/src/config/apply.rs

```diff
@@ -198,6 +202,45 @@ fn execute_inner(
     } = aziotctl_common::config::apply::run(aziot, aziotcs_uid, aziotid_uid)
         .map_err(|err| format!("{:?}", err))?;
 
+    let old_identityd_path = Path::new("/etc/aziot/identityd/config.d/00-super.toml");
+    if old_identityd_path.exists() {
+        let old_identity_config = std::fs::read(&old_identityd_path)
+            .map_err(|err| format!("could not read identity config file: {}", err))?;
+
+        let aziot_identityd_config::Settings { hostname, .. } =
```

Comment:
Failure to read or parse the old settings should probably just be ignored and treated as the old file not existing. I don't think it should fail the command.

Reply:
Addressed both comments

Reply:
The fix only addresses the file not being read. You still need to fix the file not getting parsed, which is line 208.